addon/utils/module-registry.js

```diff
@@ -1,26 +0,0 @@
-/*globals requirejs, require */
-
-import Ember from 'ember';
-
-if (typeof requirejs.entries === 'undefined') {
-  requirejs.entries = requirejs._eak_seen;
-}
-
-function ModuleRegistry(entries) {
```

Comment:
Moved this to be an ES class in the classic resolver. I would like to add a `module-registries/` directory later when adding module unification support, but the registries in that directory should be compatible with Glimmer module registries IMO, and I'd like this non-standard one (which is only used in one place AFAICT) to not live there.